tokens/light.json

```diff
@@ -388,22 +484,34 @@
             "type": "color",
             "description": "$color.rose.50",
             "failedToResolve": true
+          },
+          "medium": {
+            "value": "{color.rose.100}",
+            "type": "color",
+            "description": "$color.rose.100",
+            "failedToResolve": true
           }
         }
       }
     },
     "border": {
       "neutral": {
         "DEFAULT": {
-          "value": "{color.zinc.150}",
+          "value": "rgba({color.black}, 0.06)",
```

Comment:
Unsure about this one in tailwind

Reply:
We use it here: https://github.com/Teamtailor/design-tokens/blob/main/output/theme-light.css#L61-L64
And `border-success` seems to works! 👯‍♂️